### PR TITLE
Added Grafana public signup detection

### PR DESCRIPTION
**Pull request by ehsandeep**
### Template / PR Information

- Added Grafana public signup detection

### Template Validation

I've validated this template locally?
- [x] YES
- [ ] NO

### Additional References:

- [Nuclei Template Creation Guideline](https://nuclei.projectdiscovery.io/templating-guide/)
- [Nuclei Template Matcher Guideline](https://github.com/projectdiscovery/nuclei-templates/wiki/Unique-Template-Matchers)
- [Nuclei Template Contribution Guideline](https://github.com/projectdiscovery/nuclei-templates/blob/master/.github/CONTRIBUTING.md)
- [PD-Community Discord server](https://discord.gg/projectdiscovery)

### CODE REVIEW

**Review by toufik-airane — APPROVED** [review]
Awesome! We reach a new level of fingerprinting by registering a fake user.
Do you think it could have legal/ethical consequences? I just want to challenge the narrative because, for example, Shodan.io clearly states that it doesn't try to sign in or sign up.

For example, do you think specifying a hardcoded domain like "nuclei-test-case.com" to help organisations to clear the accounts database?

File: misconfiguration/grafana-public-signup.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+requests:
+  - raw:
+      - |
+        POST /api/user/signup/step2 HTTP/1.1
+        Host: {{Hostname}}
+        content-type: application/json
+        Origin: {{BaseURL}}
+        Referer: {{BaseURL}}
+
+        {"username":"{{randstr}}@tld","password":"{{randstr_1}}"}
```
Comment (toufik-airane) [inline misconfiguration/grafana-public-signup.yaml:18]: Do you think we could hardcode a domain to help organisation clear their accounts database after scanning?

**Comment by toufik-airane** [comment]
We could do the same with the Jira `sign up` endpoint. 👀

**Comment by ehsandeep** [comment]
@toufik-airane thank you.

We are open to updating the template if we find another safe way to confirm whether signup is enabled or disabled. Even though the form is public, you will receive an error upon registration stating that signup is disabled, so we are open to updating the template if we find another safe way to confirm the same.

File: misconfiguration/grafana-public-signup.yaml (hunk 1 above)
Comment (ehsandeep) [reply misconfiguration/grafana-public-signup.yaml]: I've added `nuclei_` as unique prefix, following by random string, as the username needs to be unique each time, we can not hardcode a static one.

**Review by toufik-airane — APPROVED** [review]
Great!

**Comment by ehsandeep** [comment]
@toufik-airane I've also added `intrusive` tag for identification/exclusion as discussed here - projectdiscovery/nuclei#551

**Pull request merged**